src/utils/localtime.ts

```diff
@@ -1,3 +1,6 @@
+import graph from '@/graph';
```

Comment:
License header should be at the first line.

Reply:
> License header should be at the first line.

ok